common/menu-entry.c

```diff
@@ -490,6 +490,13 @@ bool menuEntryLoad(menuEntry_s* me, const char* name, bool shortcut) {
         return false;
     }
 
+    //check for .filename.star in same path
+    strptr = getSlash(me->path);
+    if (strptr[0] == '/') strptr++;
+    snprintf(tempbuf, sizeof(tempbuf)-1, "%.*s.%.*s.star", (int)((strlen(me->path)) - (strlen(strptr))), me->path, (int)(strlen(strptr)), strptr);
```

Comment:
There are some redundant parentheses in this statement. I think `strlen(strptr)` should be stored in a temp var instead of being called twice.

Reply:
Done.